library/Icingadb/Widget/ViewModeSwitcher.php

```diff
@@ -95,7 +119,7 @@ public function setViewModeParam($viewModeParam)
      */
     public function getViewMode()
     {
-        return $this->url->getParam($this->getViewModeParam(), static::DEFAULT_VIEW_MODE);
+        return $this->url->getParam($this->getViewModeParam(), $this->getDefaultViewMode());
```

Comment:
Once you're populating the form, you should use this instead:

```suggestion
        return $this->getValue($this->getViewModeParam(), $this->getDefaultViewMode());
```